compose/cli/docopt_command.py

```diff
@@ -21,7 +24,9 @@ def sys_dispatch(self):
         self.dispatch(sys.argv[1:], None)
 
     def dispatch(self, argv, global_options):
-        self.perform_command(*self.parse(argv, global_options))
+        with warnings.catch_warnings():
+            warnings.filterwarnings(action='ignore', category=InsecureRequestWarning)
+            self.perform_command(*self.parse(argv, global_options))
```

Comment:
I'm not sure about this. I think this warning can be raised in other situations (when the host doesn't have the proper certs package installed) and we'd be masking those errors. 

I think I'd prefer to just keep warning.